src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation_Expected.sarif

```diff
@@ -21,65 +20,76 @@
+          "SARIF1013": {
+            "id": "SARIF1013",
+            "name": {
+              "text": "EndColumnMustNotBeLessThanStartColumn"
+            },
+            "shortDescription": {
+              "text": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property."
+            },
+            "fullDescription": {
+              "text": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property."
+            },
+            "messageStrings": {
+              "Default": "{0}: The value of the \"endColumn\" property is {1}, which is less than the value of the \"startColumn\" property, which is {2}."
+            },
+            "richMessageStrings": {},
+            "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.0/csprd01/sarif-v2.0-csprd01.html"
+          }
         }
       },
       "files": {
-        "file:///G:/Code/sarif-sdk/src/Microsoft.CodeAnalysis.Sarif.Multitool/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation.sarif": {
-          "mimeType": "application/octet-stream"
+        "file:///C:/Code/sarif-sdk/src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation.sarif": {
+          "mimeType": "application/sarif-json"
         }
       },
-      "invocation": {
-        "commandLine": "\"G:\\Code\\sarif-sdk\\bld\\bin\\SarifCli\\AnyCPU_Debug\\SarifCli.exe\"  validate G:\\Code\\sarif-sdk\\src\\Microsoft.CodeAnalysis.Sarif.Multitool\\TestData\\EndColumnMustNotBeLessThanStartColumn\\EndColumnLessThanStartColumnInRelatedLocation.sarif -o G:\\Code\\sarif-sdk\\src\\Microsoft.CodeAnalysis.Sarif.Multitool\\TestData\\EndColumnMustNotBeLessThanStartColumn\\EndColumnLessThanStartColumnInRelatedLocation_Expected.sarif -j G:\\Code\\sarif-sdk\\src\\Sarif\\Schemata\\Sarif.schema.json ",
-        "startTime": "2016-08-29T23:11:40.876Z",
-        "endTime": "2016-08-29T23:11:41.222Z",
-        "processId": 12580,
-        "fileName": "G:\\Code\\sarif-sdk\\bld\\bin\\SarifCli\\AnyCPU_Debug\\SarifCli.exe",
-        "workingDirectory": "G:\\Code\\sarif-sdk\\bld\\bin\\SarifCli\\AnyCPU_Debug"
-      }
+      "invocations": [
+        {
+          "startTime": "2018-07-25T16:16:39.516Z",
+          "endTime": "2018-07-25T16:16:40.219Z",
+          "toolNotifications": [
+            {
+              "id": "MSG001.AnalyzingTarget",
+              "physicalLocation": {
+                "fileLocation": {
+                  "uri": "file:///C:/Code/sarif-sdk/src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation.sarif"
+                }
+              },
+              "message": {
+                "text": "Analyzing 'EndColumnLessThanStartColumnInRelatedLocation.sarif'..."
+              },
+              "level": "note",
+              "time": "2018-07-25T16:16:39.813Z"
```

Comment:
>2018-07-25T16:16:39.813Z [](start = 23, length = 24)

will this work? i.e., if you emit time data in your baseline will it match a generated actual?

Reply:
The unit tests do a "selective compare".

---
In reply to: [205217627](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205217627) [](ancestors = 205217627)

Reply:
got it. the problem here though is that the presence of this data will result in unwanted changes every time we rerun your refresh scripts.

---
In reply to: [205217627](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205217627) [](ancestors = 205217627)